lib/statham_logger.ex

```diff
@@ -47,25 +47,32 @@ defmodule StathamLogger do
       a confirmation is received.
 
   ## Implementing sanitization for new types can be done in 2 ways:
```

Comment:
in two ways, but there are 3 described. Let's note, that 3rd type is not preferred